ext/standard/config.m4

```diff
@@ -455,7 +455,7 @@ PHP_NEW_EXTENSION(standard, array.c base64.c basic_functions.c browscap.c crc32.
                             http_fopen_wrapper.c php_fopen_wrapper.c credits.c css.c \
                             var_unserializer.c ftok.c sha1.c user_filters.c uuencode.c \
                             filters.c proc_open.c streamsfuncs.c http.c password.c \
-                            random.c,,,
+                            random.c hrtime.c,,,
```

Comment:
That is not a so-easy subject to takle.  Systems tend to be pretty different in their API for time measurement.
Also, you should have a read at http://nadeausoftware.com/articles/2012/03/c_c_tip_how_measure_cpu_time_benchmarking